governance/tools-transfer.md

```diff
@@ -12,12 +12,19 @@ The tomesh tools team is responsible for managing the following:
 - Server: All the files are hosted on the tomesh1 [Digital Ocean](https://www.digitalocean.com) droplet.
 - DNS: [hover](www.hover.com) is the DNS service provider. This service allows us to map tomesh.net to the IP address of the tomesh1 droplet.
 - Github: The [tomesh.net repository](https://github.com/tomeshnet/tomesh.net) hosts the website files.
+- Build A Node page is pulled from (prototype-cjdns-pi)[https://github.com/tomeshnet/prototype-cjdns-pi] README.md
```

Comment:
@darkdrgn2k #MarkdownFail :)

Reply:
Lesson of the day dont post stuff at 2am:/